src/app/task/task.component.scss

```diff
@@ -0,0 +1,28 @@
+* {
+    margin: 10px 10px 10px 10px;
+}
+
+h2 {
+    color: darkblue;
+}
+
+button {
+    padding: 0px 0px 0px 0px;
+}
+
+.add-task:hover {
```

Comment:
I am not sure if bootstrap has any classes for this kind of styling but before writing your own styles you should always checkout if bootstrap already has it.

Reply:
https://mdbootstrap.com/docs/jquery/utilities/spacing/ - there is something like it - I'll remember about your suggestion.